app/views/pre_registrations/edit.html.erb

```diff
@@ -1,132 +1,156 @@
+                    <%= f.text_field :personalEmail , :class => 'form-control', required: 'true',:maxlength=>"128", readonly: @flag ? true : false %>
+                </p>
+            </div>
 
-            </p>
-        </div>
+            <div class="col-md-6">
+                <%= f.label :holiday_scheme_id, 'Holiday Scheme' do %>
+                    <%= 'Holiday Scheme' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.select :holiday_scheme_id, options_for_select(@holiday_scheme.collect { | holiday | [holiday.name,  holiday.id] }, selected: @pre_registration.holiday_scheme_id), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
 
-        <div class="col-md-6">
-            <p>
-                <%= label :user_id, 'Name of TTF' %>
-                <%= label '', ' *', class: 'required-asterisk' %>
-            </p>
-            <p>
-                <%= f.select :user_id, options_for_select(@ttf_list.collect { | ttf_list | [ttf_list.name,  ttf_list.id] }, selected: @pre_registration.user_id), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
+                </p>
+            </div>
 
-            </p>
+            <div class="col-md-6">
+                <%= f.label :weekend_id, 'WeekEnd Scheme' do %>
+                    <%= 'Weekend Scheme' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.select :weekend_id, options_for_select(@weekend_scheme.collect { | weekend | [weekend.name,  weekend.id] }, selected: @pre_registration.weekend_id), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
+                </p>
+            </div>
         </div>
 
-        <div class="col-md-6">
-            <p>
-                <%= label :weekend_id, 'WeekEnd' %>
-                <%= label '', ' *', class: 'required-asterisk' %>
-            </p>
-            <p>
-                <%= f.select :weekend_id, options_for_select(@weekend_scheme.collect { | weekend | [weekend.name,  weekend.id] }, selected: @pre_registration.weekend_id), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
-            </p>
-        </div>
+        <div class="row">
 
-        <div class="col-md-6">
-            <p>
-                <%= label :NdaSigned, 'NDA Signed' %>
-            </p>
-            <p>
-                <%= f.check_box :NdaSigned, :checked => @pre_registration.NdaSigned %>
-            </p>
-        </div>
+            <div class="col-md-6">
+                <%= f.label :NdaSigned, 'NDA Signed' do %>
+                    <%= 'NDA Signed' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :NdaSigned, :checked => @pre_registration.NdaSigned, :required => true %>
```

Comment:
Change the hash syntax

Reply:
changed apu
please check it once again just to make sure whether I've used proper syntax or not